frontend/src/app/sessions/user-sessions-wrapper/create-session/provision-workspace/provision-workspace.component.html

```diff
@@ -0,0 +1,33 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+<div class="wrapper">
+  <h2 id="title">Provision workspace</h2>
+  <mat-card id="readonly-card">
+    <mat-card-content>
+      <p>Load the models in a training environment.</p>
+
+      <form [formGroup]="provisionForm" (ngSubmit)="requestSessions()">
+        <fieldset *ngIf="projectType !== 'training'">
+          <mat-slide-toggle formControlName="persistentWorkspace">
+            Use a persistent workspace</mat-slide-toggle
+          >
+        </fieldset>
```

Comment:
```suggestion
        <div class="mb-4" *ngIf="projectType !== 'training'">
          <mat-slide-toggle formControlName="persistentWorkspace">
            Use a persistent workspace</mat-slide-toggle
          >
        </div>
```
Resolves the sonar cloud issue and, at least from what I can see, does not change any layouting.
Additionally, I moved the `mt-4` from the button fieldset to a `mb-4` here as we only need this additional margin if this slider exists.